internal/controller/service/controller_test.go

```diff
@@ -55,25 +55,6 @@ var _ = Describe("Service Controller", func() {
 			}
 			Expect(k8sClient.Create(ctx, ns)).Should(Succeed())
 
-			By("Creating a new VirtualMachineBMC")
-			virtualMachineBMC := &virtualmachinev1.VirtualMachineBMC{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      testVirtualMachineBMCName,
-					Namespace: testVirtualMachineBMCNamespace,
-				},
-				TypeMeta: metav1.TypeMeta{
-					APIVersion: "virtualmachine.kubevirt.io/v1alpha1",
-					Kind:       "VirtualMachineBMC",
-				},
-				Spec: virtualmachinev1.VirtualMachineBMCSpec{
-					Username:                testUsername,
-					Password:                testPassword,
-					VirtualMachineNamespace: testVMNamespace,
-					VirtualMachineName:      testVMName,
-				},
-			}
-			Expect(k8sClient.Create(ctx, virtualMachineBMC)).To(Succeed())
-
```

Comment:
Why is this removed? How does the VirtualMachineBMC CR get updated without it being created first?